managed/src/main/resources/logback-sbt.xml

```diff
@@ -0,0 +1,150 @@
+<!-- https://www.playframework.com/documentation/latest/SettingsLogger -->
```

Comment:
How is this different from logback.xml? 
I checked the link but there was only mention of logback.xml nothing about logback-sbt.xml.
Maybe add some comments here explaining the purpose.

Reply:
So, the context here is:
In new sbt version it has special classloader for build processing, which has very limited set of classes in it.
And play-ebean plugin is initialised during the build, which causes logging config to be parsed and processed.
As our default logback.xml file contains this:

`        <filter class="ch.qos.logback.core.filter.EvaluatorFilter">
          <evaluator>
            <expression>
              (((String) mdc.get("logType")) == "audit") &amp;&amp;
              (${auditLogOutputToFile:-true} == false)
            </expression>
          </evaluator>
          <OnMatch>DENY</OnMatch>
        </filter>
  `
during logging config processing this expression is transformed to java code, which returns ch.qos.logback.classic.Level and is then compiled by https://janino-compiler.github.io/janino/.

But, the issue is that Janino(or Logback itself, don't rememder exactly) gets classloader for compilation process using SomeClass.class.getClassLoader() - which returns SBT internal classloader with only JVM and internal SBT classes in it - and hence an Exception is thrown during compilation that ch.qos.logback.classic.Level class not found.
  
Spent a day to figure this out and find a fix for that. :)

So, as a fix - I'm just creating special logback config for all the SBT operations and set it via .sbtopts. It's a copy of logback.xml, but without the above expression. This also affects dev mode - so decided to keep it as close to original logging config as possible to have similar logging in dev vs prod.

Reply:
Maybe say that: 
This config is used during compilation configured via .sbtOpts

Reply:
Added comment.